app/models/project.rb

```diff
@@ -1,13 +1,19 @@
 # Project class
 class Project < ActiveRecord::Base
   mount_uploader :thumbnail, ThumbnailUploader
-  translates :description, :text
   translates :subtitle, :string
+  translates :introduction, :text
+  translates :description, :text
+  globalize_accessors locales: I18n.available_locales,
+                      attributes: translated_attribute_names
 
   has_many :images
   has_many :contributions
   has_many :people, -> { distinct }, through: :contributions
 
+  # the newest projects are the most important
+  default_scope { order('year DESC') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.